include/swift/Serialization/SerializedModuleLoader.h

```diff
@@ -80,6 +80,12 @@ class SerializedModuleLoaderBase : public ModuleLoader {
     return false;
   }
 
+  /// Determines if the provided path represents a swiftmodule in either the
+  /// module cache or prebuilt module cache.
+  virtual bool isInModuleCache(StringRef DepPath) {
```

Comment:
Nitpick (can be follow-up PR): maybe instead of `isInModuleCache` we can just say `isCached`? Or something like `isCachedArtifactForDependencyPurposes` if you want to be explicit.

Reply:
Good point. It should be about how it's used rather than implemented. Updated to `isCached` and changed the doc comment to the prose equivalent of your longer name suggestion.